src/main/java/org/opensearch/flowframework/constant/CommonValue.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.constant;
+
+public class CommonValue {
+
+    public static Integer NO_SCHEMA_VERSION = 0;
+    public static final String META = "_meta";
+    public static final String SCHEMA_VERSION_FIELD = "schema_version";
+    public static final Integer GLOBAL_CONTEXT_INDEX_SCHEMA_VERSION = 1;
+    public static final String GLOBAL_CONTEXT_INDEX_MAPPING =
```

Comment:
Can we put this in `src/main/resources` and read it into this constant off the class path?  See [here](https://github.com/opensearch-project/anomaly-detection/blob/713d556d980ccb144eb6ae952cebc7b3963ed28c/src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java#L192-L194).

Reply:
I'm fine with putting this index mapping in either path, I think it's just different style of code. But I do prefer to put it in under `src/main/java` cause it can save us from setting up methods to read from `src/main/resources`, something like `getMapping()` ML-Commons is using this approach as well, see [here](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/CommonValue.java#L45)